src/main/kotlin/no/nav/klage/oppgave/api/KlagebehandlingController.kt

```diff
@@ -35,4 +49,37 @@ class KlagebehandlingController(
         return klagebehandlingFacade.getKlagebehandling(oppgaveId)
     }
 
+    @GetMapping("/oppgaver/{id}/klagebehandling/kvalitetsvurdering")
```

Comment:
Jeg skulle ønske vi slapp å gå via oppgave for disse. I mitt hode så er det ganske så midlertidig. Derfor blir jeg litt usikker på om det er verdt det. Jeg er usikker på om vi faktisk kommer til å bruke oppgave-modellen når vi tar i bruk klagebehandlinger. Snakker gjerne om det :) Er mulig jeg er litt ivrig.